sympy/solvers/solveset.py

```diff
@@ -978,6 +985,353 @@ def _solveset(f, symbol, domain, _check=False):
     return result
 
 
+def _expo_solver(f):
+    r"""
+    Helper function for solving exponential equations.
```

Comment:
I will make it: `Helper function for solving (supported) exponential equations.`